src/youtube-player/youtube-player.ts

```diff
@@ -110,6 +120,13 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy {
   }
   private _suggestedQuality = new EventEmitter<YT.SuggestedVideoQuality | undefined>();
 
+  /**
+   * If true on initialize, then the component will wait until the script is available
```

Comment:
Just need to update JsDoc to reflect the new name/behavior

nit: we typically phrase boolean flags starting with `Whether...`